hue-api/scheduledEvent.js

```diff
@@ -36,7 +36,7 @@ module.exports.create = function () {
             }
 
             if (arg.status) {
-                scedule.withEnabledState(arg.status);
+                schedule.withEnabledState(arg.status);
```

Comment:
this is basically a separate fix. @peter-murray In case you are not accepting this PR please make sure you're fixing this typo anyway. thx.